wrangler.toml

```
@@ -1,7 +1,3 @@
-name = "jt99-janitor"
-main = "src/index.ts"
-account_id = "07c634e5b9f9eb260972946dc8e2f891"
-compatibility_date = "2022-09-03"
-
-[build]
-    command = "npm run build"
\ No newline at end of file
+name = "jt99-janitor-rewrite"

```

You can keep the name the same, we don't need to keep the old worker around
```suggestion
name = "jt99-janitor"
```